pkg/util/utils.go

```diff
@@ -223,53 +218,6 @@ func MapKeysToArray(mapWithKeys map[string]interface{}) []string {
 	return result
 }
 
-// GetContainerUIDGid returns the uid and gid (and string forms) to be used running most containers
-func GetContainerUIDGid() (uidStr string, gidStr string, username string) {
```

Comment:
I moved it to `dockerutils` to avoid an import cycle when checking for rootless here.